files/tasks.py

```
@@ -413,176 +414,225 @@ def encode_media(
-                  friendly_token=media.friendly_token,
-                  action="media_auto_transcription",
-                  extra=extra_info,
-              )
-              logger.info(f"Notification sent for {friendly_token}")
-
-          return True
-      except Exception as e:
-          logger.error(f"Exception saving subtitle: {str(e)}")
-          return False
+                # Success! Keep the transcription request
+                return True
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)